### PR TITLE
AMD/Xilinx Versal ACAP: add build documentation.

### PR DESCRIPTION
**Pull request by ldts**
Add relevant documentation to allow booting a buildroot image from SD. Also document how to boot the bootable image (BOOT.BIN containing firmware/tf-a/op-tee/uboot) from JTAG.

Signed-off-by: Jorge Ramirez-Ortiz <jorge@foundries.io>

### CODE REVIEW

File: building/devices/index.rst [hunk 1]
```diff
@@ -6,7 +6,8 @@ Device specific information
 
 .. toctree::
     :maxdepth: 1
-
+	       
+    versal
```
Comment (ldts) [inline building/devices/index.rst:10]: this is described as AMD/Xilinx Versal ACAP so had to put it at the top

File: building/devices/index.rst [hunk 2]
```diff
@@ -6,7 +6,8 @@ Device specific information
 
 .. toctree::
     :maxdepth: 1
-
+
```
Comment (jbech-linaro) [inline building/devices/index.rst:9]: When looking at the `git diff` I see a lot of blank spaces on "empty" lines as well as after the last character on quite a few lines in the patch set. Please remove those (this line is one of them).

**Review by jbech-linaro — COMMENTED** [review]
Just a minor comment about empty spaces and besides that I've tried parts of the instructions locally and that worked without any issues. What it didn't try was the thing that requires Petalinux downloads. Mainly because I have no account (sure I can create one and I'll probably do that at some point, but not right now).

With the spaces fixed, please add by R-B tag.
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: building/devices/index.rst (hunk 2 above)
Comment (ldts) [reply building/devices/index.rst:9]: um sorry, my emacs settings messed up

**Comment by ldts** [comment]
@jbech-linaro I also took the opportunity to update the copyright date (funny, this is the second time I do it in two different years...am I the default "Copyright date" maintainer :) ?

**Comment by jbech-linaro** [comment]
> @jbech-linaro I also took the opportunity to update the copyright date (funny, this is the second time I do it in two different years...am I the default "Copyright date" maintainer :) ?

Thanks, please add my R-B tag for that as well, then I'll merge this PR.

**Comment by ldts** [comment]
> > @jbech-linaro I also took the opportunity to update the copyright date (funny, this is the second time I do it in two different years...am I the default "Copyright date" maintainer :) ?
> 
> Thanks, please add my R-B tag for that as well, then I'll merge this PR.

awesome. thanks!

**Pull request merged**